public/css/burger.css

```diff
@@ -29,13 +29,18 @@
   outline: none;
 }
 
+.burger-menu span,
+.burger-menu span::before,
+.burger-menu span::after {
+  height: calc(3px + 0.3em);
```

Comment:
Yep :smile: it means that there is a minimum height on the element of three pixels, and in our current use case it often will be three pixels high. It fixes some issues with numerical rounding that were causing odd appearance. This set up also allows us to retain the ability to scale the entire component by changing a single font-size number, so it can take it's dimensions from the font-size of it's containing parent component.